scripts/verify-entra-id-detector-activation-candidate.sh

```
@@ -0,0 +1,120 @@
+  "Rollback And Disable Procedure"
+  "Entra ID remains source evidence only."
+  "does not authorize direct Entra ID actioning"
+  "does not make Entra ID the authority for AegisOps case state, approval state, action state, execution state, or reconciliation outcomes"
+)
+
+for expected in "${required_candidate_text[@]}"; do
+  require_contains "${candidate_doc}" "${expected}"
+done
+
+require_contains "${microsoft_onboarding_doc}" 'Readiness state: `schema-reviewed`'
+if grep -Fqx -- 'Readiness state: `detection-ready`' "${microsoft_onboarding_doc}" >/dev/null; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)